libs/ml/src/ops/dataholder.cpp

```diff
@@ -112,6 +112,12 @@ OperationsCount DataHolder<TensorType>::ChargeBackward() const
   return cost;
 }
 
+template <typename TensorType>
+OperationsCount DataHolder<TensorType>::ChargeConstruct()
+{
+  return 1;
```

Comment:
I doubt this is correct. Constructing a DataHolder means constructing a tensor, which is more complex than 1 primitive operation. Add a constant to ml/charge_estimation (maybe called EMPTY_TENSOR_CONSTRUCTION or something like that) and use it here; i'm sure it will get reused elsewhere too

Reply:
Dataholder has pointer to Tensor and that tensor is constructed on SetData call, so it is really constant.

Reply:
ah.... this also is changing with the my other PR I think.
We can leave it for now and come back to it when we do finetuning